firecrown/ccl/systematics/lss.py

```diff
@@ -1,8 +1,9 @@
 import pyccl as ccl
+import numpy as np
 
 from ..core import Systematic
 
-__all__ = ['LinearBiasSystematic']
+__all__ = ['LinearBiasSystematic', 'MagnificationBias']
```

Comment:
We should stick with the naming convention by adding `Systematic` to the end. Or we should remove this from the linear bias. Either way they should be the same.